test/api/v01/matrix_test.rb

```diff
@@ -244,4 +244,11 @@ def test_demo_nil_quotas
       assert last_response.ok?, last_response.body
     end
   end
+
+  def test_no_content
+    post '/0.1/matrix', { api_key: 'demo', src: '48.407237, -2.816638, 44.44469, -2.75015', strict_restriction: false }
+
+    assert last_response.ok?, last_response.body
+    assert_equal [[0, 36107], [36291, 0]], JSON.parse(last_response.body)["matrix_time"]
+  end
```

Comment:
Do we need this test? I can't figure out what it exactly tests. These points are reachable (if not using Here) so we return a proper matrix.

Reply:
You are right, I forgot to specify the here mode. I added the mode and changed the assert. We need it because it cover the functionality of the unit test in `here.rb`